### ISSUE TITLE
Using Uessentials causes people on server to disconnect

### ISSUE THREAD

**Issue by Lordfool24** [uEssentials/uEssentials#368]
Ok so my server has always ran perfectly, no one has ever had a disconnect but since I installed Uessentials people tell me the game disconnects them automatically. I dont know if this has to do with my PC or the essentials but it would be a shame to have to uninstall such a great program! Is there any way to fix this? Thanks in advance!

**Comment by leonardosnt** [comment]
I can't reproduce this... if someone is having this issue too, please tell me.

**Comment by sharkbound** [comment]
i dont know if it related but  i reported something similar a while ago. Could be related, but its probly not https://github.com/uEssentials/uEssentials/issues/361

**Comment by jorea143** [comment]
http://pastebin.com/raw/vBsgvTdh

Is this related? Because I keep getting this and sometimes there's disconnection among large group of players.

**Comment by zombiepanda** [comment]
yes same here this a is a uessentilas bug it happens to be like randomly everyone will get disconnected and mabye 6-7 people will stay on its really annoying

**Comment by leonardosnt** [comment]
There is no way to confirm that this is related to uEssentials.

I've seen this happening a lot of times... on servers that doe's not use uEssentials...

**Comment by Riddler76** [comment]
Hi guys, i have this problem too. My server are on dedicated server.

I have 3 server unturned

Server configuration :
8 core
32Go RAM
300 Mbit/s

Plugins :
FeexHitman.dll
FeexRanks.dll
AutoShutdown2.dll
LPX.dll
Uconomy.dll
ZaupFeast.dll
uEssentials.dll
MessageAnnouncer.dll
Kits.dll
Votifier.dll
GlobalBan.dll
ZaupUconomyEssentials.dll

All plugins are up-to-date.

I have mass disconnecting, i think is when people die, but i'm not sure.

**Comment by visloom28vi** [comment]
i used to have this problem

**Comment by rwalkos** [comment]
This information may help some people here.

Unturned use to kick people that are in a vehicle that blows up. I think this has been fixed recently though. If you are running a windows server and you try to hold the counsel still to look at something that happens it will boot some or all of the people online. Nether of these are related to uEssintials.

The above information is just some problems I have ran across.
